rpc/harmony/v2/types.go

```diff
@@ -235,6 +235,7 @@ type StakingTxReceipt struct {
 	Type              staking.Directive `json:"type"`
 	Root              hexutil.Bytes     `json:"root"`
 	Status            uint              `json:"status"`
+	EffectiveGasPrice *big.Int          `json:"effectiveGasPrice"`
```

Comment:
What's the rationle of adding `EffectiveGasPrice` field to the StakingTxReceipt and not on the other receipt like `TxReceipt` or `CxReceipt)

Reply:
Fixed. EffectiveGasPrice assigned to TxReceipt. Thanks!